geomstats/information_geometry/binomial.py

```diff
@@ -0,0 +1,146 @@
+import geomstats.backend as gs
+import geomstats.errors
+from geomstats.geometry.base import OpenSet
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+from scipy.stats import binom
+
+
+class BinomialDistributions(OpenSet):
+    """Class for the manifold of binomial distributions."""
+
+    def __init__(self, n_draws):
+        super(BinomialDistributions, self).__init__(dim=1, ambient_space=Euclidean(dim=1), metric = BinomialFisherRaoMetric(n_draws))
+        self.n_draws = n_draws
+
+    def belongs(self, point, atol=gs.atol):
+        """Evaluate if a point belongs to the manifold of binomial distributions.
+
+        Parameters
+        ----------
+        point : array-like, shape=[...]
+            Point to be checked.
+        atol : float
+            Tolerance to evaluate positivity.
+            Optional, default: gs.atol
+
+        Returns
+        -------
+        belongs : array-like, shape=[...,]
+            Boolean indicating whether point represents a binomial
+            distribution.
+        """
+        belongs = len(point.shape) == 1
+        belongs = gs.logical_and(belongs, gs.logical_and(atol <= point, point <= 1-atol))
+        return belongs
+
+    @staticmethod
+    def random_point(n_samples=1):
+        """Sample parameters of binomial distributions.
+
+        The uniform distribution on [0, 1] is used.
+
+        Parameters
+        ----------
+        n_samples : int
+            Number of samples.
+            Optional, default: 1.
+
+        Returns
+        -------
+        samples : array-like, shape=[..., 1]
+            Sample of points representing binomial distributions.
+        """
+        return gs.random.rand(n_samples)
+
+    def projection(self, point, atol=gs.atol):
+        """Project a point in ambient space to the open set.
```

Comment:
"Project a point in ambient space to the parameter space of binomial distributions" ?